void-docs.md

```diff
@@ -0,0 +1,71 @@
+# NAME
+
+void-docs - Void Linux documentation package
+
+# SYNOPSIS
+
+*/usr/share/doc/void/html* - documentation in HTML format
+
+*/usr/share/doc/void/markdown* - documentation in Markdown format
+
+*/usr/share/doc/void/mandoc* - documentation in roff format
+
+# DESCRIPTION
+
+The `void-docs` package contains a snapshot of the online documentation from
+https://docs.voidlinux.org, which intends to document installation,
+configuration and system management for Void Linux. It is packaged in three
+formats.
+
+*/usr/share/doc/void/html* - can be viewed with any browser, such as Mozilla
+Firefox or Chromium. Recommended when a GUI session is available, because it
+allows easy navigation between the documentation pages and has the same format
+as the official website. Can be accessed by pointing the browser to the
+`/usr/share/doc/void/html/index.html` file.
+
+*/usr/share/doc/void/markdown* - can be viewed as text files, using **cat(1)**
+or **less(1)**, or as a formatted markdown file, with applications such as
+**mdcat** or **glow**. The table of contents can be accessed in the
+`/usr/share/doc/void/markdown/SUMMARY.md` file.
+
+*/usr/share/doc/void/mandoc* - can be viewed using **mandoc(1)**. Using
+**mandoc(1)** with the **-a** option, which enables a pager, is recommended.
+
+# EXAMPLES
+
+Viewing the homepage of the HTML documentation with **qutebrowser(1)**:
+
+```
+$ qutebrowser /usr/share/doc/void/html/index.html
+```
+
+Viewing the summary of the markdown documentation with **less(1)**:
+
+```
+$ less /usr/share/doc/void/markdown/SUMMARY.md
+```
+
+Viewing the Kernel page of the markdown documentation with **mdcat**:
+
+```
+$ mdcat /usr/share/doc/void/markdown/config/kernel.md
+```
+
+Viewing the Cron page of the roff documentation with **mandoc(1)**:
```

Comment:
"cron"?

Reply:
I was going by the page title in this one. Perhaps put it in quotes anyway?

Reply:
Ah, yeah, sure.

Reply:
Fixed, then :p